app/src/components/v-item-group/v-item-group.vue

```diff
@@ -24,7 +24,7 @@ export default defineComponent({
 		},
 		modelValue: {
 			type: Array as PropType<(string | number)[]>,
-			default: undefined,
+			default: null,
```

Comment:
There's a difference between `null` and `undefined` in prop default values, and we do rely on that difference at times. Are you sure these changes are correct? Feels like a footgun to just swap that out